### PR TITLE
replaced actionablemessage in Import-Token.js

### PR DESCRIPTION
**Pull request by pritam1813**
## Explanation

- This PR is Part of #19528 . 
- It replaces instance of ActionableMessage with BannerAlert in `ui/pages/swaps/import-token/import-token.js`

### Before

before

### After

after

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Hey @pritam1813, This is looking great! I have one suggestion and one optional suggestion

File: ui/pages/swaps/import-token/import-token.js [hunk 1]
```diff
@@ -5,8 +5,7 @@ import UrlIcon from '../../../components/ui/url-icon';
 import Popover from '../../../components/ui/popover';
 import Button from '../../../components/ui/button';
 import Box from '../../../components/ui/box';
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/pages/swaps/import-token/import-token.js:7]: You could also replace `UrlIcon` with `AvatarToken` from the `component-library`

[the remark this answers is not in the document]
Comment (pritam1813) [reply ui/pages/swaps/import-token/import-token.js:7]: Yeah sure, why not.This was only a single line change PR anyway. Should I try and remove all the deprecated Components in this file ?

[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/pages/swaps/import-token/import-token.js:7]: I think that would still make this PR a reasonable size to review. You could also do it in parts if you'd like to get this PR in first. Then create another PR to update the rest.

[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/pages/swaps/import-token/import-token.js:7]: I should point out that the more components that are replaced the more likely hood you will need to update e2e tests which adds a fair amount of complexity to the task.

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Thanks for your contribution
- Pulled branch checked storybook locally ✅

**Review by garrettbear — APPROVED** [review]

**Pull request merged**